src/Keyboard.cpp

```diff
@@ -209,6 +219,35 @@ size_t Keyboard_::write(const uint8_t *buffer, size_t size) {
 	return n;
 }
 
+bool Keyboard_::getLedStatus(uint8_t led)
+{
+	uint8_t _keyboardLedsStatus = HID().getKeyboardLedsStatus();
+	switch(led)
+	{
+		case LED_NUM_LOCK:
+			if (_keyboardLedsStatus & LED_NUM_LOCK) {
+				return true;
+			}
+			break;
+		case LED_CAPS_LOCK:
+			if (_keyboardLedsStatus & LED_CAPS_LOCK) {
+				return true;
+			}
+			break;
+		case LED_SCROLL_LOCK:
+			if (_keyboardLedsStatus & LED_SCROLL_LOCK) {
+				return true;
+			}
+			break;
+		default:
+			return false;
+			break;     
+	}
+	return false;    
+
+}
```

Comment:
Why not just

```c++
bool Keyboard_::getLedStatus(uint8_t led)
{
	return HID().getKeyboardLedsStatus() & led;
}
```

Reply:
Genius.

I gather some strict pedants (not me) frown on the idea of treating non-zero as a bool, rather than using an explicitly boolean comparison. 

So would prefer for example;-

```suggestion
bool Keyboard_::getLedStatus(uint8_t led)
{
	return ((HID().getKeyboardLedsStatus() & led)!=0);    
}
```

I'll give it try and confirm.

Reply:
Its good; I've tested it on my keyboard project in front, so I'll commit the suggestion. Thanks @edgar-bonet .